scripts/soroban-settings/write_upgrade_bytes/Cargo.toml

```diff
@@ -13,10 +13,10 @@ crate-type = ["cdylib"]
 doctest = false
 
 [dependencies]
-soroban-sdk = { version = "0.9.2" }
+soroban-sdk = { git = "https://github.com/stellar/rs-soroban-sdk", rev = "3f55de14f131c88e3b113f882c29a47da7b80fe2" }
```

Comment:
It probably not right to point to a git revision. I will update this to version 0.9.3 once we bump SDK version.